bin/release_branch.rb

```diff
@@ -7,10 +7,12 @@
 require 'optimist'
 
 opts = Optimist.options do
-  opt :branch, "The new branch name.", :type => :string, :required => true
+  opt :branch,        "The new branch name.",                                   :type => :string, :required => true
+  opt :source_branch, "The source branch from which to create the new branch.", :default => "master"
 
-  ManageIQ::Release.common_options(self, :except => :dry_run) # TODO: Implement dry_run
+  ManageIQ::Release.common_options(self, :except => :dry_run, :repo_set_default => nil) # TODO: Implement dry_run
 end
+opts[:repo_set] = opts[:branch] unless opts[:repo] || opts[:repo_set]
```

Comment:
This bit is a technically bug fix - previously when cutting a branch it would always use "master", even though you're supposed to create a new section.  Previously we always created the repo set as a copy of master, so it worked out, but this time, we created from lasker, so it's different, and it broke.